src/styles/main.scss

```
@@ -1,3 +1,72 @@
-body {
-  margin: 0;
+$calendar-gap: 1px;
+$cell-size: 100px;
+$calendar-width: 6 * $calendar-gap + 7 * $cell-size;
+$days:
+  "mon" 0,
+  "tue" $cell-size + $calendar-gap,
+  "wed" 2 * ($cell-size + $calendar-gap),
```

You can only calculate your margin once, instead of calculating it every time

```suggestion
  "tue" $day-margin,
  "wed" 2 $day-margin,
```